test_scripts/AppServices/SystemCapabilities/004_GetSystemCapability_unsubscribe.lua

```diff
@@ -0,0 +1,137 @@
+---------------------------------------------------------------------------------------------------
+--  Precondition: 
+--  1) Application 1 with <appID> is registered on SDL.
+--  2) Application 2 with <appID2> is registered on SDL.
+--  3) Specific permissions are assigned for <appID> with PublishAppService
+--  4) Specific permissions are assigned for <appID2> with GetSystemCapability
+--
+--  Steps:
+--  1) Application 2 sends a GetSystemCapability RPC request with subscribe = true
+--  2) Application 1 sends a PublishAppService RPC request with serviceType MEDIA
+--  3) Application 2 sends a GetSystemCapability RPC request with subscribe = false
+--  4) HMI sends ActivateAppService:false
+--
+--  Expected:
+--  1) App 2 gets a GetSystemCapability response SUCCESS
+--  2) App 1 and HMI get OnSystemCapabilityUpdated notifications
+--  3) App 2 is unsubscribed from system capability updates and does not receive OnSystemCapabilityUpdated
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/AppServices/commonAppServices')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local manifest = {
+  serviceName = config.application1.registerAppInterfaceParams.appName,
+  serviceType = "MEDIA",
+  allowAppConsumers = true,
+  rpcSpecVersion = config.application1.registerAppInterfaceParams.syncMsgVersion,
+  mediaServiceManifest = {}
+}
+
+local rpc = {
+  name = "GetSystemCapability",
+  params = {
+    systemCapabilityType = "APP_SERVICES",
+    subscribe = true
+  }
+}
+
+local expectedResponse = {
+  success = true,
+  resultCode = "SUCCESS"
+}
+
+local publishRpc = {
+  name = "PublishAppService",
+  params = {
+    appServiceManifest = manifest
+  }
+}
+
+local publishExpectedResponse = {
+  appServiceRecord = {
+    serviceManifest = manifest,
+    servicePublished = true,
+    serviceActive = true
+  },
+  success = true,
+  resultCode = "SUCCESS"
+}
+
+local function PTUfunc(tbl)
+  tbl.policy_table.app_policies[common.getConfigAppParams(1).fullAppID] = common.getAppServiceProducerConfig(1);
+  tbl.policy_table.app_policies[common.getConfigAppParams(2).fullAppID] = common.getAppServiceConsumerConfig(2);
```

Comment:
`GetSystemCapability` is not part of the `AppServiceConsumer` functional group and is already a part of `Base-4` so you probably do not need to add consumer permissions for app2